src/resolvers/teams.js

```diff
@@ -1,7 +1,26 @@
-const { sequelize } = require('../models');
-const models = require('../models');
 const retry = require('retry-as-promised');
 const { Sequelize } = require('sequelize');
+const { sequelize } = require('../models');
+const models = require('../models');
+const { sendEmail } = require('../services/sendEmail');
+
+// TO DO: Edit inivitation email subject, html, and include team name (along with link)
+const createTeamInviteMessage = teamId => {
+  // convert teamId to teamName
+  return {
+    from: 'hongyichen@uwblueprint.org',
+    subject: `Invitation to Join Building Up`,
+    html: 'You have been invited to join <strong>Building Up</strong>. Please join using this link: ',
+  };
+};
+
+const sendTeamInvites = (emails, teamId) => {
+  const message = createTeamInviteMessage(teamId);
+  emails.forEach(email => {
+    const invitationEmail = { to: { email }, ...message };
+    sendEmail(invitationEmail);
```

Comment:
it's not really a big deal, but if you wanted to write this in an "ideal" way in terms of performance you could look into `Promise.all`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

but tbh it's not the biggest deal, the `Promise.all` API can sometimes be finicky to work with